### PR TITLE
Remove unused metrics and add gauge for number of beacons in verifier…

### PR DESCRIPTION
**Pull request by bbalser**
… ready to process

### CODE REVIEW

File: iot_verifier/src/loader.rs [hunk 1]
```diff
@@ -372,9 +376,7 @@ impl Loader {
                                         report_type: ReportType::Witness,
                                         status: LoraStatus::Ready,
                                     };
-                                    metrics::increment_counter!(
-                                        "oracles_iot_verifier_loader_witness"
-                                    );
+                                    metrics.increment_witnesses();
```
Comment (andymck) [inline iot_verifier/src/loader.rs:379]: Witnesses are being incremented here but not being decremented afaics

Comment (bbalser) [reply iot_verifier/src/loader.rs:379]: That is actually to increment the witness counter, so we never need to decrement.  maybe a better function name will eliminate the confusion?

Comment (andymck) [reply iot_verifier/src/loader.rs:379]: Ah that's fine then. Reading on my phone so missed that

**Review by andymck — APPROVED** [review]

File: iot_verifier/src/metrics.rs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    pub fn count_loader_witnesses(count: u64) {
+        metrics::counter!(LOADER_WITNESS_COUNTER, count);
+    }
+
+    pub fn count_loader_invalid_witnesses(count: u64, labels: &[(&'static str, &'static str)]) {
+        metrics::counter!(INVALID_WITNESS_COUNTER, count, labels);
+    }
+
+    pub fn num_beacons(count: f64) {
+        metrics::gauge!(BEACON_GUAGE, count);
```
Comment (jeffgrunewald) [inline iot_verifier/src/metrics.rs:24]: Maybe hide the f64 requirement for gauges by moving the casting into this function? I think it’s weird enough that a gauge requires a float but now that we’re obscuring the gauge behind our own function here it looks even more arbitrary that youre tracking a gauge with a float

Comment (bbalser) [reply iot_verifier/src/metrics.rs]: Good idea, fixed 857006f

**Review by andymck — APPROVED** [review]

**Pull request merged**